lib/wikisnakker.rb

```diff
@@ -143,7 +143,7 @@ def initialize(raw)
       end]
       raw['claims'].each do |property_id, claims|
         property "#{property_id}s".to_sym do
-          claims.map { |c| Claim.new(c) }
+          claims.map { |c| Claim.new(c) }.sort { |a, b| b.rank <=> a.rank }
```

Comment:
I think this should really have a comment to make it obvious that it's relying on the fact that the ranks happen to sort in reverse alphabetical order; otherwise a future reader will likely spend too much time scratching their head at what wizardry is going on here!